maddj_recs.md

```diff
@@ -0,0 +1,7 @@
+## Recs for Los Angeles!
+
+### THINGS to DO
+* Petersen Auto Museum!
+
+### FOODS to EAT
```

Comment:
Oh, here some feedback for a super simple example that is going to be a single comment.